src/sysupdate/sysupdate-transfer.h

```diff
@@ -15,12 +15,13 @@ typedef struct Transfer Transfer;
 #include "sysupdate.h"
 
 struct Transfer {
-        char *definition_path;
```

Comment:
any reason you are dropping this? seems to be good to have for debugging, and doesn't cost much?

Reply:
It was unused, and I replaced it with an ID which is used. Seemed redundant